docs/releasenotes/0.22.md

```diff
@@ -0,0 +1,54 @@
+# actions-runner-controller v0.22.0
+
+This version of ARC focuses on scalability and reliablity of runners.
+
+## GitHub API Cache
+
+In terms of scalability, ARC now caches GitHub API responses according to their recommendation(=Cache-Control header).
+As long as GitHub keeps its current behavior, it will result in ARC to cache various List Runners API and List Workflow Jobs calls for 60 seconds.
+
+The cache for List Runners API is expecially important, as their responses can be shared between every runner under the same scope (repository, organization, or enterprise).
+
+In previous versions of ARC, the number of List Runners API calls had scaled proportional to the number of runners managed by ARC.
+Thanks to the addition of cache, since v0.22.0, it may scale proportional to the number of runner scopes (=The number of repositories for your repository runners + The number of organizations for your organizational runners + The number of enterprises for your enterprises). You might be able to scale to hundreds of runners depending on your environemnt.
```

Comment:
maybe with `+ The number of enterprises for your enterprise runners`. 😄

Reply:
Good catch 👍 Addressed in [e402b2e](https://github.com/actions-runner-controller/actions-runner-controller/pull/1199/commits/e402b2e8fd0e32598e326dbad8a3cb3d66401a5e)